kanidmd_web_ui/src/credential/reset.rs

```diff
@@ -444,20 +462,22 @@ impl CredentialResetApp {
 
                     <hr class="my-4" />
 
+                    <h4>{"Password / MFA"}</h4>
                     { pw_html }
 
                     <hr class="my-4" />
 
+                    // TODO: this could probably just be a link back home, currently it breaks and sends the user to an error..
```

Comment:
On cancel we need to send an abort to the server for the update session which also unlocks the intent token if it was used. So perhaps your server version needs update to accept that? It was cancelling correctly for me ....

Reply:
this was all tested with the current codebase 🤔

Reply:
If you click the "password and security settings" button from the dashboard to open the modal, then click cancel, it'll throw an error.